scripts/Engineering_Diffraction_2.py

```diff
@@ -0,0 +1,25 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# pylint: disable=invalid-name
+from __future__ import (absolute_import, division, print_function)
+from Engineering.gui.engineering_diffraction.engineering_diffraction import EngineeringDiffractionGui
+from qtpy import QtCore
+
+
+if 'engineering_diffraction' in globals():
```

Comment:
It's not immediately obvious why this code is here. Could you add some comments to state what this script is for?